projects/ngx-prx-styleguide/src/lib/fancy-form/fancy-field.component.ts

```diff
@@ -12,7 +12,7 @@ const isset = (val: any): boolean => val !== false && val !== undefined;
 export class FancyFieldComponent {
 
   @Input() model: BaseModel;
-  @Output() onChange = new EventEmitter<any>();
+  @Output() change = new EventEmitter<any>();
```

Comment:
How painful is this to change? It should just be in Publish right now. If it doesnt seem worth it to change this all over Publish, maybe a `// tslint:disable-next-line` is called for

Reply:
I've actually already made all of the `@Output` and selector changes in Publish. Wasn't too bad! See PRX/publish.prx.org#669